handlers/getItems.go

```diff
@@ -0,0 +1,61 @@
+package handlers
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/data"
+	"github.com/mount-joy/thelist-lambda/db"
+)
+
+type getItems struct {
+	db db.DB
+}
+
+func newGetItems() RouteHandler {
+	return &getItems{
+		db: db.NewDB(),
+	}
+}
+
+func (g *getItems) match(request events.APIGatewayProxyRequest) bool {
+	var re = regexp.MustCompile(`^/list/([\w-]+)/items/?$`)
```

Comment:
Can you add a test added that checks this regex

Reply:
You're just too eager